temporal-sdk/src/main/java/io/temporal/common/converter/DefaultDataConverter.java

```diff
@@ -82,6 +88,10 @@ public static void setDefaultDataConverter(DataConverter converter) {
   /**
    * Creates a new instance of {@code DefaultDataConverter} populated with the default list of
    * payload converters.
+   *
+   * <p>The newly created instance will use the default failure converter, configured to not encode
```

Comment:
```suggestion
   * <p>The instance is configured to not encode
```

Nitpicking, but this method returns DefaultDataConverter, so this reads strange and feels like a duplication. I think what you try to say is said on the line above: "populated with the default list of payload converters."